packages/playwright-test/src/isomorphic/teleReceiver.ts

```diff
@@ -321,6 +323,15 @@ export class TeleReporterReceiver {
     };
   }
 
+  private _parseAttachments(attachments: JsonAttachment[]): TestResult['attachments'] {
+    return attachments.map(a => {
+      return {
+        ...a,
+        body: a.body ? Buffer.from(a.body, 'base64') : undefined,
```

Comment:
Cannot use `Buffer` in isomorphic code. Let's check/test how this behaves in UI mode?

Reply:
Let's add an eslint rule